src/documents/articles/a-quick-polymer-introduction.html.md

```diff
@@ -0,0 +1,160 @@
+
+All major browsers have committed support for the v1.0 of the Shadow DOM specification, v1.0 of custom elements and HTML 5 templates.
+
+* [Safari](https://webkit.org/blog/6017/introducing-safari-technology-preview/)
+* [Internet Explorer](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/shadowdom)
+* [Firefox](https://hacks.mozilla.org/2015/11/an-update-on-web-components-and-firefox/)
+
+## HTML imports
+
+The sticking point is HTML imports. Chrome has a native implementation, but [other browsers](http://stackoverflow.com/questions/21177267/es6-modules-vs-html-imports) are looking to Javascript ES6 modules (with a polyfill for HTML imports).
+
+## webcomponents.js
+
+Browser support is varying (at the time of writing) so [webcomponents.js](https://github.com/webcomponents/webcomponentsjs) was made to polyfill the web components API.
+
+This code provides all you need to get started with web components!
+
+# What is Polymer JS?
+
+There are a few wrappers around web components such as [x-tag](http://x-tag.readme.io/) and [bosonic](https://bosonic.github.io).
+
+[Polymer](https://www.polymer-project.org) is a similar wrapper with a few extra goodies thrown in on top.
+
+The aim of the Polymer project is not to create a bloated library, but instead make the smallest library possible, and make the browser do all the work.
+
+\#UseThePlatform is the name Polymer give to represent the want for the browser to do all the heavy lifting without libraries.
+
+One of the advantages of using Polymer is that it has a built in data binding model (like [React](https://facebook.github.io/react/) or [Angular](http://angularjs.org/)) enabling complete applications to be made without any other libraries.
+
+Polymer [supports evergreen browsers](https://www.polymer-project.org/1.0/docs/browsers) and when combined with webcomponents.js supports IE 11+.
+
+# A quick Polymer example
+
+
+    <dom-module id="my-element">
+      <template>
+        <style>
+        .box{
+          color: grey;
+        }
+        </style>
+        <div class="box">{{mybinding}}</div>
+        </template>
+        <script>
+        // element registration
+        Polymer({
+          is: "my-element",
+
+          properties: {
+            mybinding: {
+              type: String,
+              value: "My text here"
+            }
+          }
+          });
+        </script>
+    </dom-module>
+
+The example above creates a new component <code>my-element</code> and it has one binding called <code>mybinding</code>. This can be changed programatically or with attribute binding:
+
+    <my-element id="obj" mybinding="New text"></my-element>
+
+    this.$.obj.set('mybinding','Even newer text');
+    //this.$ is a Polymer representation of local DOM nodes
+
+# PRPL pattern
+
+PRPL (pronounced purple) is a set of concepts to enable the next generation of web applications:
```

Comment:
Please explain how this relates to Polymer (and Web Components in general as well as Progressive Web Applications if it makes sense)